Issue #6 - Fix intltool build

hfiguiere opened the pull request:
This should address the build issue.

cc @ndim 

Close #6

hfiguiere committed:
Issue #6 - Fix intltool build

ndim commented:
Can't test this right now. Fedora stopped shipping libexif-gtk some time ago, so I need to compile and install more than just gtkam to build gtkam. I can test this in a few days.

hfiguiere commented:
it builds without.

ndim committed:
Add m4/ directory to repo because intltoolize writes to it

intltoolize requires that the m4/ directory exists, so we add
m4/ to the git repo.

ndim commented:
gtkam `configure` aborts when called with either of the following parameter sets:

    --without-libexif-gtk
    --without-libexif-gtk --without-libexif
    --without-libexif
